integration-tests/jenkins-setup/plugins.txt

```diff
@@ -0,0 +1,3 @@
+ssh-slaves:latest
+greenballs:latest
```

Comment:
Btw we should not need this for the tests.
It only changes the success color from blue to green.